### PR TITLE
Added Makefile for YUV PVR demos

### PR DESCRIPTION
**Pull request by gyrovorbis**
- Looks like we forgot to make a directory-level Makefile for the new YUV PVR examples
- Added directory-level Makefile to top-level PVR example Makefile

### CODE REVIEW

File: examples/dreamcast/pvr/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # KallistiOS ##version##
 #
 # examples/dreamcast/pvr/Makefile
-# (c)2002 Megan Potter
+# (c) 2002 Megan Potter
```
Comment (ljsebald) [inline examples/dreamcast/pvr/Makefile:4]:
```suggestion
# Copyright (C) 2002 Megan Potter
```

File: examples/dreamcast/pvr/yuv_converter/Makefile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# KallistiOS ##version##
+#
+# examples/dreamcast/pvr/Makefile
```
Comment (ljsebald) [inline examples/dreamcast/pvr/yuv_converter/Makefile:3]:
```suggestion
# examples/dreamcast/pvr/yuv_converter/Makefile
```

File: examples/dreamcast/pvr/yuv_converter/Makefile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# (c) 2023 Andy Barajas
```
Comment (ljsebald) [inline examples/dreamcast/pvr/yuv_converter/Makefile:4]: The copyright here shouldn't be Andy's, as you're the one adding the file. Either it's yours or nobody's, unless he sent you the file outside of here somehow. If that's the case (or it should be you instead), please fix the formatting to match the suggestion on the other Makefile.

**Review by ljsebald — CHANGES_REQUESTED** [review]

File: examples/dreamcast/pvr/yuv_converter/Makefile (hunk 3 above)
Comment (andressbarajas) [reply examples/dreamcast/pvr/yuv_converter/Makefile:4]: @gyrovorbis Put your name there

Comment (ljsebald) [reply examples/dreamcast/pvr/yuv_converter/Makefile:4]:
```suggestion
# Copyright (C) 2023 Falco Girgis
```

**Review by ljsebald — APPROVED** [review]

**Pull request merged**